deploy/crds/apps.openshift.io_servicebindingrequests_crd.yaml

```diff
@@ -17,7 +17,8 @@ spec:
     status: {}
   validation:
     openAPIV3Schema:
-      description: Expresses intent to bind an operator-backed service with a Deployment
+      description: ServiceBindingRequest Expresses intent to bind an operator-backed
+        service with a Deployment
```

Comment:
Not necessarily a `Deployment` is it? `applicationSelector` is more general.. it could e.g. be a Knative Serving service.

Reply:
Good catch, thank you. I've renamed it to "application workload"